### ISSUE TITLE
Donation Banner

### ISSUE THREAD

**Issue by emmawedekind** [Coding-Coach/coding-coach#149]
Add a donation banner to the landing page to allow the community to help fund the hosting, logo creation, etc.

Code for the button:
```
<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
<input type="hidden" name="cmd" value="_s-xclick">
<input type="hidden" name="hosted_button_id" value="3LA9RLHEXNZV6">
<input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif" border="0" name="submit" alt="PayPal - The safer, easier way to pay online!">
<img alt="" border="0" src="https://www.paypalobjects.com/de_DE/i/scr/pixel.gif" width="1" height="1">
</form>
```

**Comment by aswinsekar** [comment]
I'll do this

**Comment by emmawedekind** [comment]
Super! I'll assign you and post the designs here once they're ready!

**Comment by aswinsekar** [comment]
- @emmawedekind   The button code seems to be missing, Update it once!

**Comment by emmawedekind** [comment]
It's in the description: 
![image](https://user-images.githubusercontent.com/7671983/46999398-defe4100-d125-11e8-8da8-bcdfaebad5c4.png)

@aswinsekar Did you want me to un-assign you from this given our conversation?

**Comment by tureey** [comment]
Is there any Idea though about where to place this banner @emmawedekind ? If the task is un-assigned for any reason, I would love to take it. Just saying.

*PS: I guess this `code snippet` was generated by you in Paypal? If that's the case, I think there is some code missing because there is a closing `</form>` tag but is not opened.*

**Comment by tureey** [comment]
Since **we need this as soon as possible**, I am going to start with it. **Sorry** @aswinsekar .
**If you read this**, as soon as possible text me and we can talk about it and I can let you finish if I didn't by then.

**Comment by crysfel** [comment]
@emmawedekind how/where are we going to display the button? I think we should add another section in the home page asking for donations.

**Comment by emmawedekind** [comment]
Maybe in the footer on the left side? Let me put together a quick mock up tonight!

**Comment by emmawedekind** [comment]
Thinking we can just add the button to the footer like this. Obviously the background-color would match the app!
![image](https://user-images.githubusercontent.com/7671983/47229989-615a5f80-d3c9-11e8-9e75-44e9372a18a8.png)

**Comment by emmawedekind** [comment]
Closing this as it was fixed by #153
